caas/kubernetes/clientconfig/client.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package clientconfig
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+
+	"github.com/juju/errors"
+)
+
+func newK8sClientSet(config *clientcmdapi.Config, contextName string) (*kubernetes.Clientset, error) {
```

Comment:
I don't think we need this new function here.
It's only used in one place - to get a cluster client so that ensureJujuAdminServiceAccount() can be run. I think it better just to have this method as a helper in the plugin.go file along with ensureJujuAdminServiceAccount? I think that's better than leaving droppings over the code base that are not used anywhere expect in one spot.

Reply:
sure, will move back to plugin.go